assigner/config/schemas.py

```diff
@@ -0,0 +1,282 @@
+
+SCHEMAS = [
```

Comment:
I worry this would get unruly fast. It's less rigid but would make more sense to define INITIAL_SCHEMA, make that version 0/1, then copy it and modify what changed in two as part of the script. Then you only need to read blocks of code to determine changes rather than hunting for comments.

Reply:
You've got `copy.deepcopy` if you agree ;).

Reply:
I fear that writing diff functions rather than schemas will make it harder to write schemas (the proposed version 2 schema required some troubleshooting) and that it'll slow down startup -- if we could `constexpr` all of this that would be great...

Reply:
That's fine with me. I just feel there's a better way to orgfanize this. Maybe different files for each schema version?

Reply:
Yeah, we can do that!